### DISCUSSION TITLE
Buy or sell version 61 rev 4.51

### DISCUSSION THREAD

**Discussion by THorst92** [nliaudat/esp32_8ch_motor_shield#72] [Buy or Sell]
Hello all,

I have ordered 5 pieces and can sell 3 from germany as  soon as they arrive.

**Comment by MrAges** [comment]
Hey,

i´m interested in 3 pieces. Are they still available? Are they assambled and do you ship to austria?

**Reply by THorst92** [reply]
Hi,

they are still available.
I hope i recieve them in two weeks.
They are perassembled except the pinheads for the ESP32.
You can contact me via mail: stuermisch.piepmatz0c@icloud.com.

**Comment by Ceglarek10** [comment]
> Hallo zusammen,
> 
> Ich habe 5 Stück bestellt und kann 3 aus Deutschland verkaufen, sobald sie eintreffen.

Hallo. Ich werde 1 stk kaufen

**Reply by THorst92** [reply]
Hi, aktuell sind leider alle bereits reserviert.

**Comment by nliaudat** [comment]
plz update your firmware to latest version as IA/IB motor control is inverted
https://github.com/nliaudat/floor-heating-controller/releases
